compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -11851,9 +11851,9 @@ OMR::Z::TreeEvaluator::arraycmplenEvaluator(TR::Node * node, TR::CodeGenerator *
 
    if (TR::isJ9() && !comp->getOption(TR_DisableSIMDArrayCompare) && cg->getSupportsVectorRegisters())
       {
-      // An empirical study has showed that CLC is faster for all array sizes if the number of bytes to copy is known to be constant
+      // An empirical study has showed that CLC is faster for all array sizes if the number of bytes to compare is known to be constant
       if (!node->getChild(2)->getOpCode().isLoadConst())
-         return TR::TreeEvaluator::arraycmpSIMDHelper(node, cg, NULL, NULL, true, !node->isArrayCmpSign()/*return102*/, true);
```

Comment:
arraycmplenEvaluator returns the index of the first mismatch therefore return 102 flag has no impact. isArrayCmpSign() checks the opcode and only accepts 'arraycmp' but this is 'arraycmplen' and causes error.